src/index.js

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
 import './index.css';
-import App from './App';
+import { createStore } from 'redux';
+import App from './components/App';
+import rootReducer from './reducers/index';
+
+const store = createStore(rootReducer);
```

Comment:
- [ ] [REQUIRED] Based on project requirements, Books example to initial state object should be added from src/index.js. [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/InitialBooks.png)